README.md

```diff
@@ -22,36 +53,26 @@ OPTIONS:
             Print version information
 ```
 
-Check:
+
+## Check the API
+
+Ensure that the explorer backend is functioning correctly by executing the following command:
 
 ```bash
-> curl http://localhost:4000
+> curl http://localhost:4000/api/v1
 Welcome to Iroha 2 Block Explorer!
 ```
 
-## Deploy
-
-- [Install Rust](https://www.rust-lang.org/tools/install)
-- Build binary:
-
-  ```bash
-  cargo build --release
-  ```
+## API
 
-- Prepare Iroha Client config ([reference](https://github.com/hyperledger/iroha/blob/ea45b5053018acd48340024800786ff5a3d0904d/docs/source/references/config.md)). **Define target peer location here**.
+Refer to [Block Explorer API](api.md).
 
-- Run web server:
 
-  ```bash
-  ./target/release/iroha2_explorer_web \
-      -c /path/to/client_config.json \
-      -p 8080 # may be env PORT, default is 4000
-  ```
+## Tools
 
-## API
+The following tools are available in conjunction with the Iroha 2 blockchain explorer:
 
-Refer to [Block Explorer API](api.md).
+- [genesis-gen](./tools/genesis-gen/README.md): Genesis generator (a tool to generate sample data).
+- [explorer-deploy-dev-tool](https://github.com/0x009922/explorer-deploy-dev-tool): A tool for automating the deployment of Iroha and the explorer.
```

Comment:
I'd suggest making it a table like this:

|Tool|Purpose|
|--|--|
|[genesis-gen](./tools/genesis-gen/README.md)|Generate sample data|